VTUL/data-repo
# temporarily disable doi requests

**Disables doi requests until application code can be modified to work with Datacite API.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1777) (:star:)

# What does this Pull Request do? (:star:)
Disables doi requests until application code can be modified to work with Datacite API.

# What's the changes? (:star:)
* Comment out the button that triggers call to Datacite API to mint doi

# How should this be tested?
* place `user_list.txt` and `admin_list.txt` in "ansible/local_files/" directory with your vt email added to each file. (examples attached below) This will grant you admin privileges.
* build application using LIBTD-1777 branch
* log in to application
* Upload a file and add metadata
* Create a Collection and add the file to it
* On the Dashboard page click the "Publish" tab
* Select the Collection you created and click "Request DOI"
* From the login dropdown menu click "Manage DOI requests"
* Check to make sure that the "Mint DOI" button does NOT show up under the "Actions" column

# Additional Notes:
* branch: `LIBTD-1777`

[user_list.txt](https://github.com/VTUL/data-repo/files/3118443/user_list.txt)
[admin_list.txt](https://github.com/VTUL/data-repo/files/3118444/admin_list.txt)


# Interested parties
@yinlinchen 

(:star:) Required fields
